dev-tools/reconfigurator-cli/src/main.rs

```diff
@@ -76,13 +81,66 @@ struct ReconfiguratorSim {
     /// External DNS zone name configured
     external_dns_zone_name: String,
 
+    /// RNG for collection IDs
+    collection_id_rng: TypedUuidRng<CollectionKind>,
+
     /// Policy overrides
     num_nexus: Option<u16>,
 
     log: slog::Logger,
 }
 
 impl ReconfiguratorSim {
+    fn new(log: slog::Logger) -> Self {
+        Self {
+            system: SystemDescription::new(),
+            collections: IndexMap::new(),
+            blueprints: IndexMap::new(),
+            internal_dns: BTreeMap::new(),
+            external_dns: BTreeMap::new(),
+            silo_names: vec!["default-silo".parse().unwrap()],
```

Comment:
This isn't important but was the change from `"example-silo"` to `"default-silo"` significant?

I think I went with `"example-silo"` originally to emphasize that this was a toy setup, not intended to mirror a specific real system and where this silo doesn't necessarily correspond to The Default Silo.

Reply:
Definitely a brain fart from me, sorry!

Reply:
Fixed.